github/repos.go

```diff
@@ -894,11 +896,21 @@ type AdminEnforcedChanges struct {
 	From *bool `json:"from,omitempty"`
 }
 
+// LockBranch represents if the branch is marked as read-only. If this is true, users will not be able to push to the branch.
+type LockBranch struct {
+	From *bool `json:"lock_branch"`
+}
+
```

Comment:
Please delete these lines.

Reply:
Just commited a deletion for both struct